Automattic/sensei
# Tweaks to "Enrolled Courses" column

### Changes proposed in this Pull Request
This PR makes the following changes to the _Enrolled Courses_ column:
- Removes the "X Courses Enrolled" text as per https://a8c.slack.com/archives/C013QUH20TS/p1650652689256969.
- The more link was glitchy when clicked. Now it just shows the additional courses when clicked, similar to the more link for modules on the Courses page.
- Displays "N/A" if a learner is not enrolled in any courses.

### Testing instructions
- Check that the "X Courses Enrolled" text has been removed from the _Enrolled Courses_ column.
- Test that clicking the more link shows any additional courses.
- Ensure that "N/A" is displayed for students who aren't enrolled in any courses.

### Screenshot / Video]
![Screen Shot 2022-04-25 at 8 43 08 AM](https://user-images.githubusercontent.com/1190420/165091338-cc2995a3-179f-4fdd-9866-ec972f223baf.jpg)
